nimble/host/src/ble_eatt.c

```diff
@@ -476,6 +476,8 @@ ble_eatt_tx(uint16_t conn_handle, uint16_t cid, struct os_mbuf *txom)
         goto error;
     }
 
+    ble_att_truncate_to_mtu(eatt->chan, txom);
```

Comment:
If this is happening in GATT/SR I would expect fix to be in ble_att_svr.c before `ble_att_tx()`

What do you think ?